### ISSUE TITLE
Database permissions error with 1.2.1

### ISSUE THREAD

**Issue by jasonmccallister** [engram-design/Workflow#75]
### Description
We are receiving the following error when upgrading to version 1.2.1.

```
One of Workflow’s migrations failed.

Database Exception: SQLSTATE[42501]: Insufficient privilege: 7 ERROR: permission denied: "RI_ConstraintTrigger_c_31309" is a system trigger
The SQL being executed was: ALTER TABLE "public"."assetindexdata" DISABLE TRIGGER ALL; ALTER TABLE "public"."assets" DISABLE TRIGGER ALL; ALTER TABLE "public"."assettransformindex" DISABLE TRIGGER ALL; ALTER TABLE "public"."assettransforms" DISABLE TRIGGER ALL; ALTER TABLE 
```

### Steps to reproduce

1. Setup the database user with only the permission defined on Craft system requirements
2. Run migration, see the above error

### Additional info

- Plugin version: 1.2.1
- Craft version: 3.2.1

**Comment by engram-design** [comment]
I've never seen that sort of error - a permission denied error. Workflow also doesn't touch anything related to the `asset` tables. No other migrations are being run?

Can you let me know what DB drive you're using, and if there are any permissions on the database that should be worth noting?

**Comment by jasonmccallister** [comment]
That was not the full error, I will send it through your website

**Comment by engram-design** [comment]
Should be fixed in [1.2.2](https://github.com/verbb/workflow/releases/tag/1.2.2).
